### PR TITLE
DOC-425 v2 Drafts for BCGSC miRNA Profiling, Birdseed, BRASS, BWA, BWA with BQSR, BWA with Mark Duplicates and BQSR and BWA-aln workflows

### PR DESCRIPTION
**Pull request by ebusygina**
Drafts for BCGSC miRNA Profiling, Birdseed, BRASS, BWA, BWA with BQSR, BWA with Mark Duplicates and BQSR and BWA-aln workflows

### CODE REVIEW

File: docs/Encyclopedia/pages/Birdseed.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Birdseed
+
+## Description ##
+
+Birdseed is a genotyping algorithm used for the detection of single nucleotide polymorphisms (SNPs) in GDC genotyping array data harmonization.
+
+## Overview ##
+
+Birdseed is utilized for genotyping SNPs from microarray data. This pipeline processes raw array data, identifies SNPs, and generates genotype calls. It is specifically designed for genotyping arrays such as the Affymetrix SNP 6.0 array, providing accurate SNP detection across different sample types.
+
```
Comment (wwysoc2) [inline docs/Encyclopedia/pages/Birdseed.md:10]: Add this:  "Birdseed files were originally processed and generated by the TCGA program and brought into the GDC Data Portal when the GDC Legacy Archive was retired."

**Review by wwysoc2 — CHANGES_REQUESTED** [review]
My comments are probably all over the place, so here's a summary:

- Skip all BWA pages for now.  I'm confused about which workflow goes with what.  We need more information here.
- I added some recommendations for changes in the other pages.

File: docs/Encyclopedia/pages/BCGSC_miRNA_Profiling.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# BCGSC miRNA Profiling
+
+## Description ##
+
+BCGSC miRNA Profiling is a miRNA profiling pipeline used in GDC for miRNA quantification analysis.
+
+## Overview ##
+
+BCGSC miRNA Profiling is a part of the miRNA Analysis Pipeline in the GDC miRNA quantification analysis. It uses tumor and normal alignments and generates Isoform Expression Quantification and miRNA Expression Quantification data.
```
[the remark this answers is not in the document]
Comment (ebusygina) [reply docs/Encyclopedia/pages/BCGSC_miRNA_Profiling.md]: changed

File: docs/Encyclopedia/pages/BCGSC_miRNA_Profiling.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+### Input
+
+* Tumor BAM - miRNA-Seq
```
[the remark this answers is not in the document]
Comment (ebusygina) [reply docs/Encyclopedia/pages/BCGSC_miRNA_Profiling.md]: changed

File: docs/Encyclopedia/pages/BRASS.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
+# BRASS
+
+## Description ##
+
+BRASS is a structural variant calling pipeline used in GDC whole genome sequencing (WGS) harmonization.
+
+## Overview ##
+
+BRASS is one of the four pipelines used for WGS variant calling at the GDC. Structural variant calling is performed with BRASS using tumor and normal alignments and generates Structural Rearrangement data.
```
[the remark this answers is not in the document]
Comment (ebusygina) [reply docs/Encyclopedia/pages/BRASS.md]: removed

File: docs/Encyclopedia/pages/BWA-aln.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
+# BWA-aln
+
+## Description ##
+
+BWA-aln is a sequencing alignment and quality improvement workflow used in the GDC miRNA-Seq data harmonization.
```
[the remark this answers is not in the document]
Comment (ebusygina) [reply docs/Encyclopedia/pages/BWA-aln.md]: changed for later + deleted BWA for now

File: docs/Encyclopedia/pages/BWA-aln.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+BWA-aln is a sequencing alignment and quality improvement workflow used in the GDC miRNA-Seq data harmonization. 
+
+## Overview ##
+
+BWA-aln is a BWA based alignment workflow for miRNA-Seq. BWA-aln is used for aligning sequence reads to a reference genome for both normal and tumor tissues.
+
+### Input
+
+Submitted sequencing reads
```
[the remark this answers is not in the document]
Comment (ebusygina) [reply docs/Encyclopedia/pages/BWA-aln.md]: changed for later + deleted BWA for now

File: docs/Encyclopedia/pages/BWA_with_Mark_Duplicates_and_BQSR.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+
+BWA with Mark Duplicates and BQSR (Base Quality Score Recalibration) is a sequencing alignment and quality improvement workflow used in the GDC ATAC-Seq, targeted sequencing, whole genome sequencing (WGS) and whole exome sequencing (WXS) data harmonization. 
+
+## Overview ##
+
+BWA with Mark Duplicates and BQSR is a BWA based alignment workflow for ATAC-Seq, targeted sequencing, whole genome sequencing (WGS) and whole exome sequencing (WXS) with additional steps of Mark Duplicates and Base Quality Score Recalibration (BQSR). BWA with Mark Duplicates and BQSR is used for aligning sequence reads to a reference genome for both normal and tumor tissues.
+
+### Input
+
+Submitted sequencing reads
```
[the remark this answers is not in the document]
Comment (ebusygina) [reply docs/Encyclopedia/pages/BWA_with_Mark_Duplicates_and_BQSR.md]: changed for later + deleted BWA for now

File: docs/Encyclopedia/pages/Birdseed.md (hunk 1 above)
Comment (ebusygina) [reply docs/Encyclopedia/pages/Birdseed.md]: added

Comment (ebusygina) [reply docs/Encyclopedia/pages/Birdseed.md]: thank you!!

File: docs/Encyclopedia/pages/BCGSC_miRNA_Profiling.md [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+
+BCGSC miRNA Profiling is a part of the miRNA Analysis Pipeline in the GDC miRNA quantification analysis. It uses tumor or normal alignments and generates Isoform Expression Quantification and miRNA Expression Quantification data.
+
+### Input
+
+* GDC-aligned BAM - miRNA-Seq
+
+### Output
+
+* TSV (Data Type: Isoform Expression Quantification, miRNA Expression Quantification)
```
Comment (wwysoc2) [inline docs/Encyclopedia/pages/BCGSC_miRNA_Profiling.md:17]: Recommendation: Change this line to "TSV/TXT" and remove the one below.  Hopefully we can get this standardized soon.

**Review by wwysoc2 — APPROVED** [review]
Approved, once you make that small TSV/TXT change.  Then feel free to merge.

**Pull request merged**